node-fs-readfile/read-any.js

```diff
@@ -0,0 +1,17 @@
+import { readFile } from 'node:fs/promises';
+
+async function readAnyFileAsync() {
+  try {
+    if (process.argv.length < 0) {
+      console.error('Usage: node yourScript.js <filename>');
```

Comment:
Nice error handling! It makes me happy!

The only problem with it is that avgv.length will never be less than 0 because it will always have 2 elements: the path to node and the path to the .js file. So your check should be `< 3`.